packages/attribute-registry-process/src/utilities/config.ts

```diff
@@ -5,8 +5,13 @@ import {
 } from "pagopa-interop-commons";
 import { z } from "zod";
 
-const AttributeRegistryConfig =
-  CommonConfig.and(ReadModelDbConfig).and(EventStoreConfig);
+const AttributeRegistryConfig = CommonConfig.and(ReadModelDbConfig)
+  .and(EventStoreConfig)
+  .and(
+    z.object({ PRODUCER_ALLOWED_ORIGINS: z.string() }).transform((c) => ({
+      producerAllowedOrigins: c.PRODUCER_ALLOWED_ORIGINS.split(","),
+    }))
```

Comment:
```suggestion
    z.object({
      PRODUCER_ALLOWED_ORIGINS: z.string().transform((origins) =>
        origins
          .split(",")
          .map((origin) => origin.trim())
          .filter(Boolean)
      ),
    })
```
I think we can modify it like this, essentially I added a `trim()`, in order to get rid of spaces, and a `.filter(Boolean)` that gets rid of empty strings

Reply:
I don't think it's needed because `PRODUCER_ALLOWED_ORIGINS` is an env var we set manually, it will not change that often. I think it will not include typos that require that kind of check.

Reply:
> I don't think it's needed because `PRODUCER_ALLOWED_ORIGINS` is an env var we set manually, it will not change that often. I think it will not include typos that require that kind of check.

I don't think it hurts to be more careful, since it's a comma separate sequence there could be a mistake, maybe a comma with no value or something else 🤷🏻‍♂️

Reply:
I agree, it can be safer

Reply:
Updated